evenOccurrences/evenOccurrences.js

```diff
@@ -12,4 +12,14 @@
 
 const evenOccurence = (arr) => {
   // Your code here.
+<<<<<<< HEAD
+  const sortedArray = arr.split('').arr.sort().arr.join('');
+    for (let i = 0; i < sortedArray.length; i++) {
```

Comment:
Since you're not nested within a function block, this for loop doesn't need to be indented, 

I love that you sorted this array first! Nice work!